common/cpp/src/google_smart_card_common/logging/hex_dumping_unittest.cc

```diff
@@ -96,4 +119,16 @@ TEST(LoggingHexDumpingTest, HexDumpUnknownSizeInteger) {
             HexDumpUnknownSizeInteger(std::numeric_limits<int64_t>::min()));
 }
 
+TEST(LoggingHexDumpingTest, HexDumpBytes) {
+  EXPECT_EQ(HexDumpBytes(nullptr, 0), "");
+
+  const uint8_t kArray[] = {1, 2, 123};
+  EXPECT_EQ(HexDumpBytes(kArray, 3), "0x01 0x02 0x7B");
+}
+
+TEST(LoggingHexDumpingTest, HexDumpBytesVector) {
+  EXPECT_EQ(HexDumpBytes({}), "");
+  EXPECT_EQ(HexDumpBytes({1, 2, 123}), "0x01 0x02 0x7B");
```

Comment:
Parameters to EXPECT_EQ seem to be in the wrong order here

Reply:
Do you think the inverse order would be more helpful?
There was a discussion about the inverse ("Yoda-style") order in one of groups, and seems that people aren't supportive of it: https://groups.google.com/a/google.com/g/chromium-dev-internal/c/W7nPEr5Y0R8/m/i8qsXPg8DAAJ

Reply:
Correction: This seems right, but the rest of the file has it reversed :)

Reply:
Agreed, let me fix it in the same CL.

Reply:
Done.